PinvokeAnalyzer_UWPApis.txt

```diff
@@ -0,0 +1,2298 @@
+bcrypt.dll!BCryptCloseAlgorithmProvider
```

Comment:
Why did we have to add these @JohnChen0? Are these new additions or were they present earlier as well?